pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "surrealdb"
+version = "0.1.0"
+description = "SurrealDB driver for Python"
+# TODO: Add an author
+authors = []
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+aiohttp = "^3.8.1"
```

Comment:
Consider lowering this dependency to the minimum required to reduce conflicts with other packages